packages/tradershub/src/features/cfd/components/MT5PlatformsList/MT5PlatformsList.tsx

```diff
@@ -48,14 +49,20 @@ const MT5PlatformsList = ({ onMT5PlatformListLoaded }: TMT5PlatformsListProps) =
                 </div>
             )}
             {isFetchedAfterMount &&
-                data?.map(account => {
-                    if (account.is_added)
-                        return <AddedMT5AccountsList account={account} key={`added-mt5-list-${account.loginid}`} />;
+                data?.map(MT5Account => {
+                    if (
+                        MT5Account.is_added &&
+                        MT5Account.is_virtual === activeTradingAccount?.is_virtual &&
```

Comment:
For more readability, I wanted to move 
`MT5Account.is_virtual === activeTradingAccount?.is_virtual && MT5Account.account_type === accountType` to a const but These conditions need to be here as the properties (i.e is_virtual, account_type)  are associated with `is_added`

Reply:
@mitra-deriv you can filter the MT5AccountList  based on the account_type first and then map over it here

Reply:
Alright @mitra-deriv , thanks for checking.